src/credit.ts

```diff
@@ -0,0 +1,210 @@
+import AinftBase from './ainftBase';
+import { AppCreditInfo, HttpMethod, AppWithdrawList, UserWithdrawList, WithdrawRequestList } from './types';
+
+// TODO(kriii): Objectify params?
+export default class Credit extends AinftBase {
+  createAppCredit(
```

Comment:
question: should the corresponding original code be removed from asset.ts?

Reply:
No, I think original codes should remain for backward compatibility.

Reply:
As old users can use old versions of the npm package, I think we can just remove old libraries. What do you guys think? (The ainft-server rest APIs, however need to be kept for old npm packages.) @kriii @woomurf @ehgmsdk20

Reply:
If we want to guide the users of the package to migrate to the new structure, how about doing it with the next release? For users's reference, I think it's fine to just comment out the old code. But do we need to allow them to keep using the old APIs?

Reply:
I think there is no need to leave the original code, because users are using the old version of the library and there is no problem in continuing to operate as long as the old rest apis used there remains. Rather, it seems that we have to remove the existing code so that when users upgrade the package, an error saying that the function does not exist appears and we can force migration.